src/dfcx_scrapi/tools_ml/utterance_generator_utils.py

```diff
@@ -166,9 +200,18 @@ def _generate_phrases_intent(
         return synthetic_intent_set
 
     def _generate_phrases(self, training_phrases: pd.DataFrame, set_size: int):
```

Comment:
@Hgithubacct can you re-namespace `set_size` to something else?

`set` is a datatype in Python already and it's preferred to not name an arg/variable `set_x` or `x_set` if it's not actually of type `set`. 

Perhaps try `group`, `cohort`, `dataset`, or something similar.